pkg/interface/chat/src/css/custom.css

```diff
@@ -141,4 +141,46 @@ h2 {
   .h-100-minus-48-xl {
     height: calc(100% - 48px);
   }
+}
+
+/* dark */
+
```

Comment:
Another thing to consider for a dark theme are images set at a lower opacity, perhaps around 60%.

This is another eyesight saver — nothing punches a hole in a dark theme like a bright ass image coming into view.

Reply:
Everything else seems solid tho, excited to see this live.